index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tribute Page</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- [ ] kindly remove this link as it is not used anymore!

Reply:
@hammaazarok, Your comment should always sound like a suggestion. Remember, we don't want to push a narrative, we are just giving our opinion based on our experience and knowledge.